src/MPI.jl

```diff
@@ -58,6 +58,23 @@ end
 include("consts/consts.jl")
 using .Consts
 
+module API
+    using ..MPIPreferences
+    using ..Consts
+
+    names_Consts = filter(n -> startswith(string(n), "MPI_"), names(Consts; all = true)) |> collect
+    for n in names_Consts
+        @eval $n = Consts.$n  # signatures need types and constants
+    end
+
+    include("../gen/out/api.jl")
+    for name in names(@__MODULE__; all=true)
+        name in names_Consts && continue  # only export symbols from api
+        startswith(string(name), "MPI_") && @eval export $name
+    end
```

Comment:
Do we need to export them?

Reply:
No indeed, we said they should stay in a dedicated module, will fix.